tiers/models.py

```diff
@@ -64,7 +64,11 @@ def has_tier_grace_period_expired(self):
     def time_til_tier_expires(self):
         """Pretty prints time left til expiration"""
         rd = relativedelta(self.tier_expires_at, timezone.now())
-        if abs(rd.days) != 0:
+        if abs(rd.years) != 0:
+            return "{0} year, {1} months, {2} days and {3} hours".format(abs(rd.years), abs(rd.months), abs(rd.days), abs(rd.hours))
+        if abs(rd.months) != 0:
```

Comment:
Maybe have this as `elif` to be consistent with the check below it.

Reply:
Done!